<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KleidungsTypenVerwaltung</class>
 <widget class="QDialog" name="KleidungsTypenVerwaltung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Kleidungstypen</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="table_Typ"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Hinzufügen</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Anfang&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Die erste Nummer, die an ein Kleidungsstück dieses Typs vergeben werden soll.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Anfang</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Ende&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Die letzte Nummer, die an ein Kleidungsstück dieses Typs vergeben werden soll.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Ende</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edit_Typ_Name"/>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="Button_Typ">
             <property name="text">
              <string>Hinzufügen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spin_Typ_An">
             <property name="maximum">
              <number>100000</number>
             </property>
             <property name="singleStep">
              <number>1000</number>
             </property>
             <property name="value">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="spin_Typ_En">
             <property name="maximum">
              <number>100000</number>
             </property>
             <property name="singleStep">
              <number>1000</number>
             </property>
             <property name="value">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Größen</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="table_Groesse">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Hinzufügen</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Größe</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edit_Groesse"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="Button_Groesse">
             <property name="text">
              <string>Hinzufügen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>table_Typ</tabstop>
  <tabstop>edit_Typ_Name</tabstop>
  <tabstop>spin_Typ_An</tabstop>
  <tabstop>spin_Typ_En</tabstop>
  <tabstop>Button_Typ</tabstop>
  <tabstop>table_Groesse</tabstop>
  <tabstop>edit_Groesse</tabstop>
  <tabstop>Button_Groesse</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KleidungsTypenVerwaltung</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
